api/routes.go

```diff
@@ -94,4 +94,9 @@ func RegisterRoutes() {
 		"api-bsf",
 		shared.WrapApplicationJSON(shared.WrapPermissiveCORS(apiBSFHandler)),
 	)
+
+	// API endpoint for fetching test history
+	shared.AddRoute("/api/history", "api-history",
+		shared.WrapApplicationJSON(
+			shared.WrapPermissiveCORS(testHistory)))
```

Comment:
`testHistory` should be `testHistoryHandler` to match the naming conventions of the other endpoint handlers.